docs/source/models.rst

```diff
@@ -139,48 +139,134 @@ For more information, see the :mod:`mlflow.pyfunc` documentation.
+:py:func:`save_model() <mlflow.mleap.save_model>` and 
+:py:func:`log_model() <mlflow.mleap.log_model>` methods for saving MLeap models in MLflow format, 
+but these methods do not include the ``python_function`` flavor in the models they produce.
+
+A companion module for loading MLflow models with the MLeap flavor format is available 
+in the ``mlflow/java`` package. 
+
+For more information, see :py:mod:`mlflow.spark`, :py:mod:`mlflow.mleap`, and the
+`MLeap documentation <http://mleap-docs.combust.ml/>`_.
 
 PyTorch (``pytorch``)
 ^^^^^^^^^^^^^^^^^^^^^
 
-The ``pytorch`` model flavor enables logging and loading PyTorch models. Model is completely stored in `.pth` format using `torch.save(model)` method. Given a directory containing a saved model, you can log the model to MLflow via ``log_saved_model``. The saved model can then be loaded for inference via ``mlflow.pyfunc.load_pyfunc()``. For more information, see :py:mod:`mlflow.pytorch`.
+The ``pytorch`` model flavor enables logging and loading PyTorch models.
+
+The :py:mod:`mlflow.pytorch` module defines utilities for saving and loading MLflow models with the
+``pytorch`` flavor. The :py:func:`mlflow.pytorch.save_model()` and
+:py:func:`mlflow.pytorch.log_model()` methods can be used to save PyTorch models in MLflow format;
+both of these functions use the
+`torch.save() <https://pytorch.org/docs/stable/torch.html#torch.save>`_ method to serialize PyTorch
+models. Additionally, the :py:func:`mlflow.pytorch.load_model()` method can be used to load MLflow
+models with the ``pytorch`` flavor as PyTorch model objects. Finally, models produced by
+:py:func:`mlflow.pytorch.save_model()` and :py:func:`mlflow.pytorch.log_model()` contain the
+``python_function`` flavor, allowing them to be loaded as generic Python functions for inference
+via :py:func:`mlflow.pyfunc.load_pyfunc()`.
+
+For more information, see :py:mod:`mlflow.pytorch`.
+
 
 Scikit-learn (``sklearn``)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The ``sklearn`` model flavor provides an easy to use interface for handling scikit-learn models with no
-external dependencies. It saves and loads models using Python's pickle module and also generates a valid
-``python_function`` flavor model. For more information, see :py:mod:`mlflow.sklearn`.
+The ``sklearn`` model flavor provides an easy-to-use interface for saving and loading scikit-learn
+models. The :py:mod:`mlflow.sklearn` module defines 
+:py:func:`save_model() <mlflow.sklearn.save_model>` and 
+:py:func:`log_model() <mlflow.sklearn.log_model>` functions that save scikit-learn models in
+MLflow format, using either Python's pickle module (Pickle) or CloudPickle for model serialization.
+These functions produce MLflow models with the ``python_function`` flavor, allowing them to
+be loaded as generic Python functions for inference via :py:func:`mlflow.pyfunc.load_pyfunc()`.
+Finally, the :py:func:`mlflow.sklearn.load_model()` method can be used to load MLflow models with 
```

Comment:
allowing them to be loaded as generic Python functions -> allowing you to load them as generic Python functions

Reply:
Changed to active voice!